libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -650,15 +650,15 @@ static MQTTStatus_t publishToTopic( MQTTContext_t * pContext,
                          packetId );
 }
 
-static int32_t failedRecv( NetworkContext_t * pNetworkContext,
+static int32_t failedRecv( const NetworkContext_t * pNetworkContext,
                            void * pBuffer,
                            size_t bytesToRecv )
 {
     ( void ) pBuffer;
     ( void ) bytesToRecv;
 
     /* Terminate the TLS+TCP connection with the broker for the test. */
-    ( void ) Openssl_Disconnect( pNetworkContext );
+    ( void ) Openssl_Disconnect( ( NetworkContext_t * ) pNetworkContext );
```

Comment:
This explicit cast is a temporary change and will be removed in #1159 where the `const` qualifier will be added for the `Openssl_Disconnect` function